lib/web/apiserver.go

```diff
@@ -1009,6 +1009,9 @@ func (h *Handler) bindDefaultEndpoints() {
 	// SAML IDP integration endpoints
 	h.GET("/webapi/scripts/integrations/configure/gcp-workforce-saml.sh", h.WithLimiter(h.gcpWorkforceConfigScript))
 
+	// Okta integration endpoints.
+	h.GET("/.well-known/jwks-okta", h.WithLimiter(h.jwksOkta))
```

Comment:
Should we add a test for the new endpoint? No need for anything fancy, just a happy-path would suffice.